src/app/components/moj/moj-pagination/moj-pagination-item/moj-pagination-item.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-moj-pagination-item',
+  standalone: true,
+  imports: [],
+  templateUrl: './moj-pagination-item.component.html',
+})
+export class MojPaginationItemComponent {
+  @Input() extraClasses = '';
+  @Input() isActive: boolean = false;
```

Comment:
You won't need these inputs

But adds this instead: 
@HostBinding('class') class = 'moj-pagination__item';